packages/circuit-ui/components/Table/components/TableCell/TableCell.ts

```diff
@@ -83,37 +124,20 @@ const condensedPresentationStyles = ({ role, header, condensed, theme }) =>
  * TableCell component for the Table. You shouldn't import this component
  * directly, the Table handles it
  */
-const TableCell = styled.td`
+const TableCell: FC<TableCellProps> = styled.td`
   ${baseStyles};
   ${condensedStyles};
   ${presentationStyles};
   ${hoverStyles};
   ${condensedPresentationStyles};
 `;
 
-TableCell.propTypes = {
-  /**
-   * Aligns the content of the Cell with text-align
-   */
-  align: PropTypes.oneOf(['left', 'right', 'center']),
-  /**
-   * @private Add heading styles to placeholder Cell.
-   * Handled internally
-   */
-  header: PropTypes.bool,
-  /**
-   * @private Adds active style to the Cell if it is currently hovered by
-   * sort.
-   * Handled internally
-   */
-  isHovered: PropTypes.bool,
-};
-
+/**
+ * FIXME: we don't need this data-testid (should query by role when testing)
+ * but some might rely on it in their tests, so this will be a breaking change.
+ */
```

Comment:
What I meant here is that we should remove this test-id in the next major version. Is this comment clear enough?